jms/src/main/scala/akka/stream/alpakka/jms/JmsBrowseSettings.scala

```diff
@@ -16,7 +17,8 @@ final class JmsBrowseSettings private (
     val destination: Option[Destination],
     val credentials: Option[Credentials],
     val selector: Option[String],
-    val acknowledgeMode: AcknowledgeMode
+    val acknowledgeMode: AcknowledgeMode,
+    val connectionStatusSubscriptionTimeout: scala.concurrent.duration.FiniteDuration
```

Comment:
`JmsBrowseStage` doesn't actually use `JmsSettings`, just hardcode it to zero here.
(We could drop the `extends` for Alpakka 2.0, actually.)